packages/jest-resolve-dependencies/src/index.ts

```diff
@@ -15,19 +15,11 @@ import {isSnapshotPath} from 'jest-snapshot';
  * to retrieve a list of all transitive inverse dependencies.
  */
 class DependencyResolver {
-  _hasteFS: HasteFS;
-  _resolver: Resolver;
-  _snapshotResolver: Snapshot.SnapshotResolver;
-
   constructor(
-    resolver: Resolver,
-    hasteFS: HasteFS,
-    snapshotResolver: Snapshot.SnapshotResolver,
-  ) {
-    this._resolver = resolver;
-    this._hasteFS = hasteFS;
-    this._snapshotResolver = snapshotResolver;
-  }
+    private _resolver: Resolver,
```

Comment:
why moved into the constructor? please keep it as fields

Reply:
or is this some magical auto-assign thing?

Reply:
Yes, if you declare them as `public` or `private`, they become fields. Can undo it if you want, but I think it's pretty convenient :)

Reply:
I think I prefer the other way, as that's closer to how `private` will work in real JS, and not some odd TS extension (it uses `#` instead of `private`, no `this`, but still)

https://github.com/tc39/proposal-class-fields#private-fields
https://github.com/Microsoft/TypeScript/issues/9950

Reply:
@jeysal mind fixing this?

Reply:
done